packages/backend-plugin-api/src/testUtils.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// TODO: Remove this relative import when extrating the path utilities to this package
+// eslint-disable-next-line @backstage/no-relative-monorepo-imports
+import { packagePathMocks } from '../../backend-common/src/paths';
```

Comment:
We'll need to move the path utilities at the same time, i.e. merge this and #24565, they're very tightly coupled

Reply:
What about merging the https://github.com/backstage/backstage/pull/24565 in this branch?

Reply:
Yep I think that makes sense 👍, just wanna avoid having these be merged separately into `master` as they're very tightly coupled